docs/concepts/base-catalog.md

```diff
@@ -0,0 +1,71 @@
+---
+title: What is the Base Catalog?
+description: Everything you need to know about the Base Catalog
+---
+
+# What is the Base Catalog
+
+The base Catalog is the root of all dependencies in Flox.  It is generated from
+a fork of [nixpkgs][nixpkgs], contains a set of historical versions and is
+updated regularly.  When you install something with Flox, it will come from or
```

Comment:
```suggestion
updated on an automated schedule.  When you install something with Flox, it will come from or
```

I want it to be clear that this isn't an ad-hoc, manual process.